src/lib/sort-rules.js

```diff
@@ -1,9 +1,29 @@
 'use strict'
 
 function getSortedRules(rules) {
-  return rules.sort(function sort(a, b) {
-    return a > b ? 1 : -1
-  })
+  return Array.isArray(rules) ? getSortedRulesArray(rules) : transformIntoSortedRulesArray(rules)
+}
+
+function getSortedRulesArray(rules) {
+  return rules.sort(sortAlphabetically)
+}
+
+function transformIntoSortedRulesArray(allRulesArray) {
```

Comment:
I guess the name `allRulesArray` is not semantic, its and `object`, may be you want to keep it `rules`